development/build/transforms/remove-fenced-code.test.js

```diff
@@ -757,6 +805,56 @@ Always_Included
    Always_Included
 Always_Included
 
+Always_Included
+   Always_Included
+Always_Included
+   Always_Included
+Always_Included
+
+
+`,
+        true,
+      ],
+      flask: [
```

Comment:
Hah, I forgot all about these tests. This was quite strange to come across in the diff, I didn't know what I was looking at.

I was wondering why I didn't see any new fixtures for desktop, but I see now that the expected output for desktop is that all fences are preserved. So desktop is the new "base case" here rather than Flask. If we had included `ONLY_INCLUDE_IN(beta)` in any of these fixtures, we'd need to have an explicit output for each one.

Let me know if I've misunderstood what's going on here

Reply:
Exactly right, all the flask code fences stay, as desktop inherits from flask, and there was no previous beta code fences so I didn't add any either.